docs/guide/hello-world.md

```diff
@@ -168,7 +169,40 @@ export default class HelloWorld extends React.Component {
 
 If you want to learn more about our approach to writing React components, check out the [Components](../components.md) page.
 
-### 2. Hook up controller
+### 2. Create style.scss file
+
+According to [Components](../components.md) guidelines, there is only one style file per component, it's named `style.scss` and lives in the same component folder.
+
+We'll create an empty `style.scss` file with the following command:
+
+```
+touch client/my-sites/hello-world/style.scss
+```
+
+Then add some styles for our `HelloWorld` component:
+
+```
```

Comment:
We could add syntax highlighting to the code blocks here to make the code prettier on both GitHub and DevDocs 💅
Here's an example achieved by changing the opening ` ``` ` to ` ```css `:  

```css
  .hello-world__title {
    color: #37a000;
    font-size: 3rem;
 }
```
I'll drop line comments on the other blocks - if you don't mind adding those syntax tags that would be awesome 🙂